internal/clientcache/internal/db/db.go

```diff
@@ -146,7 +146,7 @@ type schema struct {
 
 const (
 	schemaTableName      = "schema_version"
-	schemaCurrentVersion = "v0.0.2"
+	schemaCurrentVersion = "v0.0.4"
```

Comment:
Do we need to bump this two patch versions?

Reply:
Probably not.  ty!